auto_gptq/modeling/_base.py

```diff
@@ -66,7 +79,9 @@ def __post_init__(self):
         fields_info = fields(self)
 
         if self.bits not in fields_info[0].metadata["choices"]:
-            raise ValueError(f"only support quantize to {fields_info[0].metadata['choices']} bits.")
+            raise ValueError(
```

Comment:
Did you apply a linter? Could you first do that in a separate PR for ease of review?

Reply:
@fxmarty 

Yes, wrote this kernel myself.  

Is there a linter / formatting rules you'd prefer?

Reply:
That's great!

Can you use `ruff==0.1.5`? https://github.com/huggingface/transformers/blob/da20209dbc26a6a870a6e7be87faa657b571b7bc/setup.py#L158

Something like `ruff . --fix`. I would merge that first, then you can apply the same linting to this PR for ease of review.

Reply:
@jeromeku https://github.com/AutoGPTQ/AutoGPTQ/pull/537 is merged. Could you install `ruff==0.1.5`, run `ruff auto_gptq examples tests setup.py --fix` on your branch, and then merge main into your branch & solve potential conflicts?